server/src/generate-schema.ts

```diff
@@ -0,0 +1,22 @@
+import { NestFactory } from '@nestjs/core';
```

Comment:
You have an auto scheme file do you need a separate entry point to create the scheme? Unless you are no committing the scheme?